exercise/templatetags/exercise.py

```diff
@@ -73,6 +74,24 @@ def _is_accessible(context, entry, t):
     return True
 
 
+@register.simple_tag
+def check_exercise_extensions(children, entry_name):
+    personal_deadline_printed = False
```

Comment:
Could this template tag be named `check_exercise_deadline_extensions`?
`check_exercise_extension` sounds like something different.

Reply:
Sure.